### PR TITLE
[stable/openebs]: update charts to 2.11.0

### PR DESCRIPTION
**Pull request by akhilerm**
- update chart version
- update readme
- update values
- update ndm daemonset template
- update ndm deployment template

Signed-off-by: Akhil Mohan <akhil.mohan@mayadata.io>

### CODE REVIEW

**Review by prateekpandey14 — COMMENTED** [review]
@akhilerm can we also update the `lvm-localpv` and `zfs-localpv` image version to the latest release version as well

**Comment by akhilerm** [comment]
Updated. @prateekpandey14

File: charts/openebs/values.yaml [hunk 1]
```diff
@@ -252,17 +251,17 @@ featureGates:
   UseOSDisk:
     enabled: false
     featureGateFlag: "UseOSDisk"
-
-crd:
-  enableInstall: true
```
Comment (prateekpandey14) [inline charts/openebs/values.yaml:257]: @akhilerm Maya-apiserver deployment uses the variable , we have to keep it.

Comment (akhilerm) [reply charts/openebs/values.yaml:257]: Done.

**Review by prateekpandey14 — APPROVED** [review]

File: charts/openebs/values.yaml (hunk 1 above)
Comment (kmova) [reply charts/openebs/values.yaml:257]: good catch @prateekpandey14 !

**Pull request merged**